src/libtrx/config/map_tr1.def

```diff
@@ -86,6 +86,7 @@ CFG_BOOL(g_Config, gameplay.enable_uw_roll, true)
 CFG_BOOL(g_Config, input.enable_buffering, false)
 CFG_BOOL(g_Config, gameplay.enable_lean_jumping, false)
 CFG_BOOL(g_Config, gameplay.enable_target_change, true)
+CFG_BOOL(g_Config, gameplay.enable_gun_flash, true)
```

Comment:
We already have `visuals.enable_shotgun_flash`, let's make it similar. For example, `visuals.enable_gun_lighting` or something. My proposal sounds poor, but the gist of it is to make these two options clearly discernible – that one is about sprites, and the other is about dynamic lighting.

I wouldn't limit this option to TR1, and would let the players also disable gun flashes and explosions in TR2.